DiegoFlores/SpringCRUD/src/main/java/com/bootcamp/vetPatitas/userController/PetController.java

```diff
@@ -0,0 +1,59 @@
+package com.bootcamp.vetPatitas.userController;
+
+import static com.bootcamp.vetPatitas.utils.Constants.PET_ADDED_SUCCESSFULLY;
+import static com.bootcamp.vetPatitas.utils.Constants.PET_DELETED_SUCCESSFULLY;
+import static com.bootcamp.vetPatitas.utils.Constants.PET_FOUNDED_SUCCESSFULLY;
+import static com.bootcamp.vetPatitas.utils.Constants.PET_UPDATED_SUCCESSFULLY;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.bootcamp.vetPatitas.dto.PetDTO;
+import com.bootcamp.vetPatitas.dto.ResponseDTO;
+import com.bootcamp.vetPatitas.service.PetService;
+import com.bootcamp.vetPatitas.utils.Constants.ResponseConstants;
+
+@RestController
+@RequestMapping("pets")
+public class PetController {
+
+	@Autowired
+	PetService petService;
+	
+	@PutMapping(path = "/savePet",produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Prefer nouns to name resources (path mapping)
https://restfulapi.net/resource-naming/